feat: Implement Windows environment compatibility with script optimization and documentation enhancements

Spock12138 opened the pull request:
## Modification Notes
1. **Windows Compatibility Adaptations**    
   - Optimized batch script execution logic
   
2. **Documentation Updates**  
   - Added database connection operations in Step 3: EXPLAIN SQL to ensure workflow continuity

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/bytedance/videx?pullRequest=5) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/bytedance/videx?pullRequest=5) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/bytedance/videx?pullRequest=5) it.</sub>

kr11 commented:
@Spock12138 

Cool! Thanks for the works on the Windows environment. I'll find a Windows environment to try the added scripts ASAP.

Additionally, please accept the license/cla to avoid delaying the PR pipeline :)

kr11 commented:
@Spock12138 
Furthermore, for global communication, could you please use English as the default language?

Spock12138 commented:
@kr11 Thanks for pointing this out! 🙏 I completely understand the importance of English for global collaboration.
I've ​​revised the PR description​​ in English with detailed context.
Checked all ​​code comments​​ and ensured they're in English.
Will maintain English as the default language in future contributions.
The changes are already pushed to this PR branch. Let me know if anything else needs adjustment.
Appreciate your patience and guidance!

kr11 commented on README.md:
````diff
@@ -178,6 +217,9 @@ If VIDEX-Server is deployed elsewhere, execute `SET @VIDEX_SERVER` first.
 
 ```sql
 -- SET @VIDEX_SERVER='127.0.0.1:5001'; -- Not needed for Docker deployment
+-- SET @VIDEX_SERVER='127.0.0.1:5001'; -- 以 Docker 启动，则不需要额外设置 
````
Looks like a duplicate line? Also, ensure that Chinese comments are not appeared in English documents. :)

Spock12138 replied:
@kr11 Thank you for the thorough review! I've implemented the suggested changes as follows:
**Redundant Code Cleanup ​**  
- Removed duplicate SQL parameter declaration in lines 218-219 (redundant SET @VIPDEX SERVER declaration)

kr11 commented:
@Spock12138 

A good document supplement! I've just added a few minor comments.

Spock12138 commented on README.md:
````diff
@@ -129,7 +145,8 @@ VIDEX-Server | 127.0.0.1:5001
 #### Step 1: Import Test Data
 
-```bash 
+#### For Linux/macOS Users：
````
@kr11 Thank you for the detailed review! I've implemented the requested changes:

**1. Header Format Standardization**  
- Changed `##### For Windows Users` to **`For Windows Users`** 
- Synchronized Chinese docs: `##### 对于 Windows 用户` → **`对于 Windows 用户`**

Spock12138 commented:
> @Spock12138
> 
> 一个很好的文档补充！我刚刚添加了一些小评论。

@kr11 Thank you for the review! I've addressed the suggested changes in [commit ](https://github.com/bytedance/videx/pull/5#discussion_r2051712268):

✅ **Implemented improvements:**
- Standardized heading hierarchy 
- Added proper spacing between sections 
- Confirmed consistent rendering in both EN/CN docs

Please let me know if you spot anything else needing adjustment. The changes are ready for your final review whenever convenient.

kr11 commented on README.md:
````diff
@@ -140,17 +157,39 @@ tar -zxf data/tpch_tiny/tpch_tiny.sql.tar.gz
  --videx 127.0.0.1:13308:videx_tpch_tiny:videx:password
 ```
 
+#### For Windows Users:
````
hi @Spock12138 let's change all `For Windows` or `For Linux/macOS` from `#### x` to `**x**`

Spock12138 replied:
@kr11 Thank you for the thorough review! I've implemented the requested changes:
- Unified documentation format from `####x` to `**x**` in `README.md`

Special thanks for catching the documentation inconsistencies - this review helped me better understand cross-platform formatting requirements. Please let me know if any further adjustments are needed 🙌

kr11 commented:
Looks good to me.
